themes/base.theme.bash

```diff
@@ -144,6 +147,12 @@ function git_prompt_vars {
     fi
   fi
 
+  if [[ "${SCM_GIT_SHOW_CURRENT_USER}" = "true" ]]; then
+    # support two or more initials, set by 'git pair' plugin
+    SCM_CURRENT_USER=`git config user.initials | sed 's% %+%'`
+    details+="$(git_user_info)"
```

Comment:
Where is this function defined? What do I need to install to have this function?